src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsForm.tag

```diff
@@ -106,12 +91,21 @@
                                 </label>
                             </h5>
                             <div class="col-sm-10">
-                                <input class="form-control" type="text"
-                                    name="<%= Const.ParamsNames.FEEDBACK_SESSION_NAME %>"
-                                    id="<%= Const.ParamsNames.FEEDBACK_SESSION_NAME %>"
-                                    maxlength=<%= FieldValidator.FEEDBACK_SESSION_NAME_MAX_LENGTH %>
-                                    placeholder="e.g. Feedback for Project Presentation 1"
-                                    value="${fsForm.fsName}">
+                                <c:choose>
+                                    <c:when test="${fsForm.fsNameEditable}">
+                                        <input class="form-control" type="text"
+                                            name="<%= Const.ParamsNames.FEEDBACK_SESSION_NAME %>"
+                                            id="<%= Const.ParamsNames.FEEDBACK_SESSION_NAME %>"
+                                            maxlength=<%= FieldValidator.FEEDBACK_SESSION_NAME_MAX_LENGTH %>
```

Comment:
We may want to just fix up these missing quotation marks in the jsp / tag files.